app/views/users/show.html.erb

```diff
@@ -0,0 +1,84 @@
+<h3><%= @user.name %></h3>
+<h4><%= @user.email %></h4>
+
+
+<h2> Created events </h2>
+
+<div class="grid size" style=" display: inline-grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  padding: 10px;
+  grid-gap: 1em;">
+<% @user.created_events.each do |event| %>
```

Comment:
[Optional] Let's follow the best practices and separate HTML file with CSS file.